apps/documentation/pages/design/components/form.mdx

```diff
@@ -110,7 +110,9 @@ This property is used to render the fields in the form. Fields can be passed as
 |             label             | `true`   | `null`  | `string`                                                                                                                            | Label for the field.                                                                                                                                                                                                      |
 |             type              | `false`  | `text`  | `text`, `date`, `number`, `email`, `password`, `tel`, `url`, `search`, `datetime-local`, `time`, `file`, `switch`, `array`, `group` | Type of the field.                                                                                                                                                                                                        |
 |           children            | `false`  | `null`  | `array`                                                                                                                             | Children fields for the field if type is array. This allows fields to be added dynamically                                                                                                                                |
+|          codeOptions          | `false`  | `null`  | `{indentation, language, blurDelay}`                                                                                                | Used to configure code editor fields, please look for docs [here](./code#properties).                                                                                                                                     |
```

Comment:
please list out the nested options and their description like was done for the fieldOptions